docs/pages/configuration/overview.md

```diff
@@ -262,6 +262,16 @@ The port on which the development server will run. Defaults to `3000`. This opti
 
 If the port is already in use, the next available port will be used.
 
+### `canonicalUrl`
```

Comment:
Maybe call this `canonicalUrlOrigin` so its clear that this is just the [URL origin](https://developer.mozilla.org/en-US/docs/Web/API/URL/origin)